$DragonFly: src/usr.sbin/tcpdump/tcpdump/print-pflog.c.patch,v 1.1 2007/10/11 02:46:51 pavalos Exp $
Index: print-pflog.c
===================================================================
RCS file: /home/dcvs/src/contrib/tcpdump-3.9/print-pflog.c,v
retrieving revision 1.1.1.3
diff -u -r1.1.1.3 print-pflog.c
--- print-pflog.c	11 Oct 2007 02:37:52 -0000	1.1.1.3
+++ print-pflog.c	11 Oct 2007 02:31:09 -0000
@@ -37,8 +37,9 @@
 #include <sys/socket.h>
 #endif
 #include <net/if.h>
-#include <net/pfvar.h>
-#include <net/if_pflog.h>
+#include <net/if_var.h>
+#include <net/pf/pfvar.h>
+#include <net/pf/if_pflog.h>
 
 
 
